ckanext/publications_qld_theme/assets/webassets.yml

```diff
@@ -0,0 +1,4 @@
+publications_qld-css:
```

Comment:
just checking, is this still 2.8 compatible?

Reply:
2.8 will ignore this file. We detect the version and use `{% resource %}` instead.